app/javascript/packages/document-capture/context/acuant.jsx

```diff
@@ -52,18 +52,36 @@ import AnalyticsContext from './analytics';
 /**
  * @typedef AcuantContextProviderProps
  *
- * @prop {string=}   sdkSrc      SDK source URL.
- * @prop {string?=}  credentials SDK credentials.
- * @prop {string?=}  endpoint    Endpoint to submit payload.
- * @prop {ReactNode} children    Child element.
+ * @prop {string=} sdkSrc SDK source URL.
+ * @prop {string?=} credentials SDK credentials.
+ * @prop {string?=} endpoint Endpoint to submit payload.
+ * @prop {number=} glareThreshold Minimum acceptable glare score for images.
+ * @prop {number=} sharpnessThreshold Minimum acceptable sharpness score for images.
+ * @prop {ReactNode} children Child element.
  */
 
+/**
+ * The minimum glare score value to be considered acceptable.
+ *
+ * @type {number}
+ */
+export const DEFAULT_ACCEPTABLE_GLARE_SCORE = 30;
```

Comment:
since these come from data attributes now, do we even need this constant? We'd only use it in tests right?

Reply:
> since these come from data attributes now, do we even need this constant? We'd only use it in tests right?

Right, basically they exist only avoid duplication for the two references in this file, plus the tests. Just leaving them scattered as magic numbers is an option, but it doesn't sit too well with me. Alternatively we could avoid some of the duplication by...

1. Getting the default value via rendering the hook in a `before` lifecycle of the tests

```js
let glareThreshold, sharpnessThreshold;
before(() => {
  const { result } = renderHook(() => useContext(AcuantContext));
  ({ glareThreshold, sharpnessThreshold } = result);
});
```

2. Create a different constant to store the default context and reference that in props defaulting

```diff
diff --git a/app/javascript/packages/document-capture/context/acuant.jsx b/app/javascript/packages/document-capture/context/acuant.jsx
index f4d445f52..2c7236c8b 100644
--- a/app/javascript/packages/document-capture/context/acuant.jsx
+++ b/app/javascript/packages/document-capture/context/acuant.jsx
@@ -62,17 +62,3 @@ import AnalyticsContext from './analytics';
 
-/**
- * The minimum glare score value to be considered acceptable.
- *
- * @type {number}
- */
-export const DEFAULT_ACCEPTABLE_GLARE_SCORE = 30;
-
-/**
- * The minimum sharpness score value to be considered acceptable.
- *
- * @type {number}
- */
-export const DEFAULT_ACCEPTABLE_SHARPNESS_SCORE = 30;
-
-const AcuantContext = createContext({
+const DEFAULT_CONTEXT = {
   isReady: false,
@@ -82,6 +68,8 @@ const AcuantContext = createContext({
   credentials: /** @type {string?} */ (null),
-  glareThreshold: DEFAULT_ACCEPTABLE_GLARE_SCORE,
-  sharpnessThreshold: DEFAULT_ACCEPTABLE_SHARPNESS_SCORE,
+  glareThreshold: 30,
+  sharpnessThreshold: 30,
   endpoint: /** @type {string?} */ (null),
-});
+};
+
+const AcuantContext = createContext(DEFAULT_CONTEXT);
 
@@ -94,4 +82,4 @@ function AcuantContextProvider({
   endpoint = null,
-  glareThreshold = DEFAULT_ACCEPTABLE_GLARE_SCORE,
-  sharpnessThreshold = DEFAULT_ACCEPTABLE_SHARPNESS_SCORE,
+  glareThreshold = DEFAULT_CONTEXT.glareThreshold,
+  sharpnessThreshold = DEFAULT_CONTEXT.sharpnessThreshold,
   children,
```

I think either of the above could be fine, though I personally don't see a huge benefit over how it's currently implemented.

Reply:
oh I was just thinking we could move the constants to the spec files, but ya I don't feel strongly enough that we need go with all the changes in (1) or (2)